gazebo_dev/package.xml

```diff
@@ -1,5 +1,6 @@
 <?xml version="1.0"?>
-<package format="2">
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
```

Comment:
Can't we keep format 2?

Reply:
We could, but format 3 [offers some advantages](http://www.ros.org/reps/rep-0149.html) and will likely be supported for longer. It seems to be the default for new ROS2 packages

Reply:
In general, I think it's best not to upgrade unless there's an explicit reason to, so we keep compatibility with older tools.

It doesn't look to me that we're using anything specific to version 3 here, so I don't see a need for the move. But it doesn't look like it's breaking anything either, so we can leave it as 3 for as long as we don't see a problem with it.